MonoGame.Framework/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -124,5 +129,68 @@ DepthStencilView IRenderTarget.GetDepthStencilView()
             GenerateIfRequired();
             return _depthStencilView;
         }
+
+        protected internal override SampleDescription CreateSampleDescription()
+        {
+            return this.GraphicsDevice.GetSupportedSampleDescription
+                (SharpDXHelper.ToFormat(this._format), this.MultiSampleCount);
+        }
+
+        internal void ResolveSubresource()
+        {
+            GraphicsDevice._d3dContext.ResolveSubresource(this._texture, 0, _resolvedTexture._texture, 0,
```

Comment:
Should I also add null and/or IsDisposed checks for `_texture` & `_resolvedTexture`, before calling `_d3dContext.ResolveSubresource()` method? I am not sure if there can be any scenario where null/disposed can happen, but I also don't want to add unnecessary checks.